### PR TITLE
SDK roll out to RC5 for OSS

### PR DESCRIPTION
**Pull request by harini-venkataraman**
## What
PR to roll out RC5 for OSS
...

## Why
This is to stabilize staging and roll out to latest RC, before declaring GA.
...

## How
Version bumps in respective services
...

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)
RC5, is not tested with tools and not bumped to the piece of it.
...

## Database Migrations
Not applicable.
...

## Env Config
Not applicable.
...

## Relevant Docs
Not applicable.
...

## Related Issues or PRs

-

## Dependencies Versions
Not applicable.
...

## Notes on Testing
Happy path of prompt studio is tested

...

## Screenshots
Build success with Release candidates
image

Execution of same prompt works file
Screenshot from 2024-12-04 12-15-55

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: backend/prompt_studio/prompt_studio_core_v2/models.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ class CustomTool(DefaultOrganizationMixin, BaseModel):
     def delete(self, organization_id=None, *args, **kwargs):
         # Delete the documents associated with the tool
         if not check_feature_flag_status(FeatureFlag.REMOTE_FILE_STORAGE):
-            file_path = FileManagerHelper.handle_sub_directory_for_prompt_studio(
+            file_path = FileManagerHelper.handle_sub_directory_for_tenants(
```
Comment (gaya3-zipstack) [inline backend/prompt_studio/prompt_studio_core_v2/models.py:143]: Was this a bug and we are reverting to the code that existed earlier?

**Review by gaya3-zipstack — APPROVED** [review]
Loooks good.

File: backend/prompt_studio/prompt_studio_core_v2/models.py (hunk 1 above)
Comment (harini-venkataraman) [reply backend/prompt_studio/prompt_studio_core_v2/models.py:143]: @gaya3-zipstack `handle_sub_directory_for_prompt_studio` is to be invoked only when feature flag is off. It should not have been added in the positive flow of the flag. Right now, this causes error while deleting prompt studio. Hence made this change.

Comment (gaya3-zipstack) [reply backend/prompt_studio/prompt_studio_core_v2/models.py:143]: ok.

**Pull request merged**